.bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "",
+    "strip_prefix": "",
+    "url": "https://github.com/buildbuddy-io/rules_xcodeproj/releases/download/{TAG}/rules_xcodeproj.{TAG}.tar.gz"
```

Comment:
This isn't the release archive. Should probably be `https://github.com/buildbuddy-io/rules_xcodeproj/releases/download/{TAG}/release.tar.gz` instead?